src/vishakha/Asgnmt33/Prgm1.java

```diff
@@ -0,0 +1,24 @@
+/*Program 1: return a reserve number.
+input : 1234
+output : 4321*/
+
+package vishakha.Asgnmt33;
+
+public class Prgm1 {
+	
+	void reverseNum(int num){
+		int numRev = 0;
+		while(num !=0){
+			int n = num % 10;
+			numRev = numRev * 10 + n;
+			num = num/10;
+		} 
+		System.out.println("Reverse No : " +numRev);
```

Comment:
I guess expectation is like to check whether both are palindrome or not
I hope you got point
if(num==numRev)